src/System.Collections.Concurrent/ref/System.Collections.Concurrent.csproj

```diff
@@ -5,6 +5,7 @@
     <AssemblyVersion>4.0.10.0</AssemblyVersion>
```

Comment:
Well since the last change I tried there was copying something here, that's like Pooh and Piglet hunting Woozles and following their own footprints around a tree.